### PR TITLE
Changes in container has been described in upgrade doc

### PR DESCRIPTION
**Pull request by shahariaazam**
Changes in container has been described and developers can easily identify the changes regarding using container in Slim

### CODE REVIEW

File: docs/v4/start/upgrade.md [hunk 1]
```diff
@@ -39,6 +39,35 @@ $app = new App(...);
 
 ## Changes to Container
 Slim no longer has a Container so you need to supply your own. If you were relying on request or response being in the container, then you need to either set them to a container yourself, or refactor. Also, `App::__call()` method has been removed, so accessing a container property via `$app->key_name()` no longer works.
+```php
+/**
+ * In Slim 3.x version container was available.
```
Comment (l0gicgate) [inline docs/v4/start/upgrade.md]: `Slim 3.x shipped with the Pimple container implementation and enabled the following synthax`

File: docs/v4/start/upgrade.md [hunk 2]
```diff
@@ -39,6 +39,35 @@ $app = new App(...);
...
+
+//Assign dependencies as array
+$container['view'] = function (\Psr\Container\ContainerInterface $container){
+    return new \Slim\Views\Twig('');
+};
+
+
+/**
+ * Slim 4.x didn't come with a container. Slim supports containers that implement PSR-11 like PHP-DI.
+ * To install DI Slim bridge `composer require php-di/slim-bridge`
```
Comment (l0gicgate) [inline docs/v4/start/upgrade.md]: Don’t recommend Slim Bridge but PHP-DI instead.

File: docs/v4/start/upgrade.md [hunk 3]
```diff
@@ -39,6 +39,35 @@ $app = new App(...);
...
+$container = $app->getContainer();
+
+//Assign dependencies as array
+$container['view'] = function (\Psr\Container\ContainerInterface $container){
+    return new \Slim\Views\Twig('');
+};
+
+
+/**
+ * Slim 4.x didn't come with a container. Slim supports containers that implement PSR-11 like PHP-DI.
```
Comment (l0gicgate) [inline docs/v4/start/upgrade.md]: `Slim 4.x doesn’t come with a container.`

**Review by l0gicgate — CHANGES_REQUESTED** [review]

File: docs/v4/start/upgrade.md (hunk 1 above)
Comment (shahariaazam) [reply docs/v4/start/upgrade.md]: Has been changed

File: docs/v4/start/upgrade.md (hunk 2 above)
Comment (shahariaazam) [reply docs/v4/start/upgrade.md]: Got it. Done!

File: docs/v4/start/upgrade.md (hunk 3 above)
Comment (shahariaazam) [reply docs/v4/start/upgrade.md]: Edited.

**Review by shahariaazam — COMMENTED** [review]
All changes have been made

File: docs/v4/start/upgrade.md [hunk 4]
```diff
@@ -39,6 +39,35 @@ $app = new App(...);
...
+
+//Assign dependencies as array
+$container['view'] = function (\Psr\Container\ContainerInterface $container){
+    return new \Slim\Views\Twig('');
+};
+
+
+/**
+ * Slim 4.x doesn’t come with a container. Slim supports containers that implement PSR-11 like PHP-DI.
+ * To install DI Slim bridge `composer require php-di/php-di`
```
Comment (l0gicgate) [inline docs/v4/start/upgrade.md]: ~~To install DI Slim bridge~~

To install PHP-DI

File: docs/v4/start/upgrade.md [hunk 5]
```diff
@@ -39,6 +39,35 @@ $app = new App(...);
...
+$container = $app->getContainer();
+
+//Assign dependencies as array
+$container['view'] = function (\Psr\Container\ContainerInterface $container){
+    return new \Slim\Views\Twig('');
+};
+
+
+/**
+ * Slim 4.x doesn’t come with a container. Slim supports containers that implement PSR-11 like PHP-DI.
```
Comment (l0gicgate) [inline docs/v4/start/upgrade.md]: ~~Slim 4.x doesn’t come with a container. Slim supports containers that implement PSR-11 like PHP-DI.~~

Slim 4.x does not ship with a container library. It supports all PSR-11 implementations such as PHP-DI

**Review by l0gicgate — CHANGES_REQUESTED** [review]

File: docs/v4/start/upgrade.md (hunk 5 above)
Comment (shahariaazam) [reply docs/v4/start/upgrade.md]: Changes done!

File: docs/v4/start/upgrade.md (hunk 4 above)
Comment (shahariaazam) [reply docs/v4/start/upgrade.md]: Changes done!

**Review by l0gicgate — APPROVED** [review]

**Pull request merged**